stubs/Mockery.php

```diff
@@ -1,4 +1,5 @@
 <?php
+// phpcs:disable PSR1.Classes.ClassDeclaration.MultipleClasses
```

Comment:
Since multiple classes/interfaces are defined in this file, just ignoring that php-cs rule for this file (but still processing other rules, which is what the rest of the changes in this file are)